packages/extensions/venia-sample-backends/intercept.js

```diff
@@ -29,56 +35,61 @@ const fetchBackends = async debug => {
 };
 
 /**
- * Validation function to check if the backend being used is one of the sample backends provided
- * by PWA Studio. If yes, the function validates if the backend is active. If not, it reports an
- * error by calling the onFail function. In the error being reported, it sends the other sample
- * backends that the developers can use.
+ * A validation function that verifies that the backend used is active if it is
+ * one of the sample backends provided by Magento. If the backend is not active
+ * the function provides a list of alternatives.
  *
- * @param {Object} config.env - The ENV provided to the app, usually avaialable through process.ENV
+ * @param {Object} config.env - The ENV provided to the app, usually available through process.ENV
  * @param {Function} config.onFail - callback function to call on validation fail
  * @param {Function} config.debug - function to log debug messages in console in debug mode
  *
- * To watch the debug messages, run the command with DEBUG=*runEnvValidators*
+ * To watch the debug messages, run the command with DEBUG=pwa-buildpack:runEnvValidators
  */
 const validateSampleBackend = async config => {
-    console.warn(
-        "\n venia-sample-backends is a Dev only extension, please remove it from the project's package.json before going to production. \n"
```

Comment:
I removed this message. The original intent of [my request](https://github.com/magento/pwa-studio/pull/2853#issuecomment-736123153) was that we document in the `Getting Started` or `FAQ` that this extension should be removed from your package.json dependencies outside of development. Now that we only validate sample backends, it shouldn't be as big of a deal, but every person who forgets to remove it will be inadvertently hitting the `getSampleBackends` endpoint for no reason.

Reply:
I feel we should have something yelling at them to remove the package, like a warning. We do the same with Braintree token but that's at a different level than this validation. What does the team think?

Reply:
Personally I don't want to be reminded that we have this extension installed on the core repo.

A solution that only installs the extension after scaffolding would suffice.